src/frontend/src/styles/main.css

```diff
@@ -933,6 +933,7 @@ by all browsers (FF is missing) */
 }
 
 .c-card__close {
+  --scale: 1;
```

Comment:
don't know well the css of two but is `--scale` name a bit generic and therefore a higher chance to be overwritten? should it be something like `--c-scale`?

Reply:
We don't have that many variables, I think `--scale` is fine for now